src/main/resources/data/integrated_circuit/advancements/recipes/root.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What is this for? I assume it has a use since minecraft:recipes/root exist, but wouldn't it be possible to use that as the parent for the IC recipes?